bindings/c/test/mako/mako.c

```diff
@@ -52,44 +77,42 @@ fdb_error_t wait_future(FDBFuture *f) {
 }
 
 
-int commit_transaction(FDBTransaction *transaction, mako_stats_t *stats) {
+int commit_transaction(mako_args_t *args, FDBTransaction *transaction, mako_stats_t *stats) {
```

Comment:
I am a bit confused: don't you have a function and a macro that basically do the same thing? Can you get rid of either of them?

Reply:
Which macro are you referring?  Maybe I had one before, but I don't think I do anymore.

Reply:
By the way, just in case you missed, this new retry logic does NOT retry the exact same transaction (with the same keys & values).  The idea is to keep the backoff states so that FDB can hopefully handle things better.  Why we don't replay the exact same transaction is for simplicity / I'm being lazy, and for benchmarking purpose, it's not a big deal if we skip failed transactions anyway.